### DISCUSSION TITLE
I need help with a student developer pack offer

### DISCUSSION THREAD

**Discussion by DaniCenter** [community/community#55217] [GitHub Education]
### Hi everyone,

Hi, I am a programming student, I study software engineering and artificial intelligence and I would like to access the benefits of github education. I have tried many times to access, however they always reject my application, and for many reasons, I would like someone to validate my documents, sometimes I try to send them from my cell phone, I try to take pictures from my main camera but I do not know why it does not let me, then I try to upload images but they tell me that they are not sufficient evidence because they can be modified, I do not know what else to do, if someone could review my documents would see that if I accredit, maybe my institution does not end with edu. pe or something like that, but I have many more documents like carnet, payment slips, portal of the institution..., please help me.

**Comment by samus-aran** [comment]
Hi @DaniCenter  👋

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
